doc/user/assorted_guide.rst

```diff
@@ -1,5 +1,82 @@
+**********************
 Physics Coupling
+**********************
+
+Loose Coupling
 ----------------
-Tight coupled physics can be activated though the ``tightCoupling``
-setting. This is handled in the :py:meth:`mainOperate <armi.operators.Operator.mainOperate>`
-method.
+ARMI supports loose and tight coupling. Loose coupling is interpreted as one-way coupling between physics for a single time node. For example, a power distribution in cycle 0 node 0 is used to calculate a temperature distribution in cycle 0 node 0. This temperature is then used in cycle 0 node 1 to compute new cross sections and a new power distribution. This process repeats itself for the lifetime of the simulation. 
+
+.. image:: images/looseCouplingIllustration.png
```

Comment:
I don't see these images in the PR. I assume they are generated during the doc build?

Reply:
https://github.com/terrapower/armi/pull/1033/commits/7a6fa1cb8cf7e8f6622c33b1d703aae7c73721e7

I will make an issue to make these via graphviz instead of a force added png.